lib/src/rules/avoid_print.dart

```diff
@@ -41,6 +42,17 @@ class _Visitor extends SimpleAstVisitor {
 
   _Visitor(this.rule);
 
+  _validateArgument(Expression expression) {
+    if (expression is SimpleIdentifier) {
+      final Element element = expression.staticElement;
+      if (element is FunctionElement &&
+          element?.name == 'print' &&
```

Comment:
since you just verified `element is FunctionElement` I think you can drop the `?`, simplifying this to:

    element.name == 'print'

Reply:
Removed unwanted null checks.